### ISSUE TITLE
[Bug]: При использовании метода `VKWebAppShowWallPostBox` с параметром `upload_attachments` для загрузки фотографии по прямому URL не работает публикация на стену

### ISSUE THREAD

**Issue by suprunchuk** [VKCOM/vk-bridge#573]
### Описание

При использовании метода `VKWebAppShowWallPostBox` с параметром `upload_attachments` для загрузки фотографии по прямому URL не работает публикация на стену в веб-версии. Загрузка работает корректно в клиентах и мобильной версии VK, но не в веб-версии. Это баг VK Bridge, так как веб-версия правильно блокирует публикацию поста.

### Версия vk-bridge

-

### Платформа

Web

### Браузер и версия

Chrome 129.0.6668.101

### Шаги воспроизведения

1. Открыть приложение/игру в веб-версии VK которое еще **не опубликовано в каталоге.**
2. Попытаться опубликовать пост на стену с фотографией, используя VKWebAppShowWallPostBox с параметром upload_attachments
3. Указать прямой URL фотографии для загрузки
4. Попытаться опубликовать пост

### Ожидаемое поведение

Пост не должен публиковаться на стену пользователя с мобильных клиентов с прикрепленной фотографией, загруженной по указанному URL.

### Скриншоты

![img-2024-10-11-12-05-46](https://github.com/user-attachments/assets/42f332ad-294b-4a63-813f-fb3baf5060b8)

**Comment by nshvyryaev** [comment]
Добрый день!
Верно я понял из описания, что проблема в том, что пост можно опубликовать на мобильных платформах?

Вижу "Ожидаемое поведение: Пост не должен публиковаться"

**Comment by suprunchuk** [comment]
@nshvyryaev Да. С мобильного телефона публикация работает на стену, а с ПК не работает. По факту она не должна ни там ни там не работать же, так как приложение еще не вышло в каталог, верно?

**Comment by nshvyryaev** [comment]
@suprunchuk Добрый день!

Принято решение открыть этот метод и для приложений, еще не попавших в каталог. Будем править документацию и проверять Web версию.

**Comment by suprunchuk** [comment]
@nshvyryaev
Добрый день. Правильно ли я понял, что репорст на стену теперь будет работать на всех платформах до публикации мини аппа в каталог?

**Comment by nshvyryaev** [comment]
А разве он был закрыт? Не работал только параметр upload_attachments. Если передавать картинки через attachments - ограничений не было.

**Comment by suprunchuk** [comment]
> А разве он был закрыт? Не работал только параметр upload_attachments. Если передавать картинки через attachments - ограничений не было.

да, это и имею ввиду. спасибо
